pkg/instrumenter/beyla.go

```diff
@@ -1,4 +1,4 @@
-package components
+package instrumenter
```

Comment:
shoud we rename this file too?

Reply:
Oh yeah! Still few "Beylas" leaked around the code 😅

Reply:
yeah i can imagine...but we can rename here to obi :smile: